deploy/gk-deploy

```diff
@@ -162,13 +162,9 @@ while [[ $# -ge 1 ]]; do
     keypos=1
     while [[ $keypos -lt $keylen ]]; do
       case ${key:${keypos}} in
-        o*|-openshift)
-        CLI="oc"
-        if [ "$key" == "--openshift" ]; then keypos=$keylen; fi
-        ;;
```

Comment:
It's a cleanup. I forgot to remove it earlier when I closed off openshift support and will currently break things if used. Hence, fix.